gapic-generator-java/src/test/java/com/google/api/generator/gapic/composer/grpc/goldens/samples/bookshopclient/AsyncGetBook.golden

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2024 Google LLC
```

Comment:
Looks like running `mvn fmt:format` resulted in all these changes. Perhaps these changes could be made in another PR and then can be removed from this PR.

Reply:
Thanks! with the latest merge of main, these changes have been removed from this PR.